ironfish-rust/src/transaction/mod.rs

```diff
@@ -111,6 +111,12 @@ impl ProposedTransaction {
         }
     }
 
+    // return the sender of the transaction
+    // TODO(joe): change to calculate via randomness with circuit update
```

Comment:
This will actually come from transaction.sender rather than calculated by randomess. Leaving comment since it will be removed in next PR.